web.sh

```diff
@@ -1 +1 @@
-gunicorn -b 0.0.0.0:$PORT app:app
+gunicorn -b 0.0.0.0:$PORT --worker-class eventlet -w 1 app:app --timeout 500
```

Comment:
Please add a comment above containing an explanation and link to documentation regarding `worker-class`